lib/identity_doc_auth/acuant/responses/get_results_response.rb

```diff
@@ -62,62 +57,28 @@ def pii_from_doc
 
         attr_reader :http_response
 
-        def generate_errors
-          return {} if successful_result?
-
-          dpi_threshold = config.dpi_threshold&.to_i || 290
-          sharpness_threshold = config&.sharpness_threshold&.to_i || 40
-          glare_threshold = config&.glare_threshold&.to_i || 40
-
-          front_dpi_fail, back_dpi_fail = false
-          front_sharp_fail, back_sharp_fail = false
-          front_glare_fail, back_glare_fail = false
-
-          processed_image_metrics.each do |side, img_metrics|
-            hdpi = img_metrics['HorizontalResolution'] || 0
-            vdpi = img_metrics['VerticalResolution'] || 0
-            if hdpi < dpi_threshold || vdpi < dpi_threshold
-              front_dpi_fail = true if side == :front
-              back_dpi_fail = true if side == :back
-            end
-
-            sharpness = img_metrics['SharpnessMetric']
-            if sharpness.present? && sharpness < sharpness_threshold
-              front_sharp_fail = true if side == :front
-              back_sharp_fail = true if side == :back
-            end
-
-            glare = img_metrics['GlareMetric']
-            if glare.present? && glare < glare_threshold
-              front_glare_fail = true if side == :front
-              back_glare_fail = true if side == :back
-            end
-          end
-
-          return { results: [Errors::DPI_LOW_BOTH_SIDES] } if front_dpi_fail && back_dpi_fail
-          return { results: [Errors::DPI_LOW_ONE_SIDE] } if front_dpi_fail || back_dpi_fail
-
-          return { results: [Errors::SHARP_LOW_BOTH_SIDES] } if front_sharp_fail && back_sharp_fail
-          return { results: [Errors::SHARP_LOW_ONE_SIDE] } if front_sharp_fail || back_sharp_fail
+        def response_info
+          @response_info ||= create_response_info
+        end
 
-          return { results: [Errors::GLARE_LOW_BOTH_SIDES] } if front_glare_fail && back_glare_fail
-          return { results: [Errors::GLARE_LOW_ONE_SIDE] } if front_glare_fail || back_glare_fail
+        def create_response_info
+          alerts = processed_alerts
 
-          #if no image metric errors default to raw error output
-          error_messages_from_alerts
+          {
+            vendor: 'Acuant',
+            doc_auth_result: result_code.name,
+            processed_alerts: alerts,
+            alert_failure_count: alerts[:failed]&.count.to_i,
+            image_metrics: processed_image_metrics,
+            raw_alerts: raw_alerts,
+            raw_regions: raw_regions,
```

Comment:
Do we need to return these two? Are they being used anywhere? Or is it just being brought over from the previous `to_h` to retain it as part of the `merge(response_info)`?

Reply:
Those were added and maintained for fallback purposes in case something weird came up in the processing of stuff. We could look at the raw outputs and hopefully learn something from the more complete data. I think I want to continue to capture them for now but  they are very verbose for the logs and are probably less needed once we are satisfied we're not  missing major  pieces of providing good errors to users.